src/objective-c/GRPCClient/GRPCCall+ChannelCredentials.h

```diff
@@ -33,26 +33,24 @@
 
 #import "GRPCCall.h"
 
-/** Helpers for setting TLS Trusted Roots, Client Certificates, and Private Key
- */
+/** Helpers for setting TLS Trusted Roots, Client Certificates, and Private Key */
 @interface GRPCCall (ChannelCredentials)
 
 /**
- * Use the provided @c pemRootCert as the set of trusted root Certificate
- * Authorities for @c host.
+ * Use the provided @c pemRootCert as the set of trusted root Certificate Authorities for @c host.
  */
 + (BOOL)setTLSPEMRootCerts:(nullable NSString *)pemRootCert
                    forHost:(nonnull NSString *)host
-                     error:(NSError *_Nullable *_Nullable)errorPtr;
+                     error:(NSError * _Nullable * _Nullable)errorPtr;
```

Comment:
The plan is to revert to without modification, then another PR to clean the format of things in GRPCClient after I've figured out how to correctly configure it. I made several try of that and result was not good